Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/UDPClient.cs

```diff
@@ -101,7 +101,7 @@ public virtual void Send(Receivers receivers = Receivers.Server, int messageGrou
 		/// <param name="natHost">The NAT server host address, if blank NAT will be skipped</param>
 		/// <param name="natPort">The port that the NAT server is hosting on</param>
 		/// <param name="pendCreates">Immidiately set the NetWorker::PendCreates to true</param>
-		public void Connect(string host, ushort port = DEFAULT_PORT, string natHost = "", ushort natPort = NatHolePunch.DEFAULT_NAT_SERVER_PORT, bool pendCreates = false, ushort overrideBindingPort = DEFAULT_PORT + 1)
+		public bool Connect(string host, ushort port = DEFAULT_PORT, string natHost = "", ushort natPort = NatHolePunch.DEFAULT_NAT_SERVER_PORT, bool pendCreates = false, ushort overrideBindingPort = DEFAULT_PORT + 1)
```

Comment:
Maybe there should be a return false after the connectAttemptFailed callback call? :thinking:

Reply:
I don't think so since the exception is thrown. 
https://github.com/BeardedManStudios/ForgeNetworkingRemastered/blob/12b7034dcb61679a322a787c1e8532431705c0ec/Forge%20Networking%20Remastered%20Unity/Assets/Bearded%20Man%20Studios%20Inc/Scripts/Networking/Forge/Networking/UDPClient.cs#L160

If you want a clean return, then yes.

I just put the bool also for UDPClient.Connect(...) just for consistency (even if the signatures are different between TCP & UDP clients - which, in my opinion, should be the same and be part of the IClient interface; different discussion :) ).

Reply:
yeah, the tcp client needs to be brought inline with what is in the udp one.